spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validation' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:email) }
+    it { is_expected.to validate_presence_of(:password) }  
+  end
+
+  describe 'error messages' do
+    it 'gives an error message when email is not provided' do
```

Comment:
- [ ] You already have the same test above, `shoulda_matchers` handles all this logic for you without writing it up.